docs/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "handsontable-documentation",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Handsontable documentation",
+  "main": "index.js",
+  "author": "Handsoncode <hello@handsontable.com>",
+  "repository": {
+    "url": "https://github.com/handsontable/handsontable",
+    "directory": "docs"
+  },
+  "scripts": {
+    "docs:lint": "eslint --ext .js,.vue .vuepress next",
+    "docs:lint:fix": "eslint --ext .js,.vue .vuepress next --fix",
+    "docs:assets:next": "node .vuepress/tools/copy-assets.mjs",
+    "docs:start": "npm run docs:assets:next && vuepress dev ./",
+    "docs:start:no-cache": "npm run docs:assets:next && vuepress dev --no-cache ./",
+    "docs:build": "npm run docs:build:no-check-links && npm run docs:check-links",
+    "docs:build:no-check-links": "vuepress build -d .vuepress/dist/docs ./",
+    "docs:docker:build": "npm run docs:docker:build:staging",
+    "docs:docker:build:staging": "npm run docs:assets:next && docker build -f docker/Dockerfile -t docs-md ../",
+    "docs:docker:build:production": "docker build -f docker/Dockerfile -t docs-md --build-arg BUILD_MODE=production ../",
+    "docs:version": "cd .vuepress/tools/version && node rollup-a-version.js",
+    "docs:api": "cd .vuepress/tools/jsdoc-convert && node jsdoc.js",
+    "docs:check-links": "node .vuepress/tools/check-links.js"
+  },
+  "license": "CC BY 4.0",
```

Comment:
Isn't this field for software license, not content license?

Reply:
IMHO It's for the package so our docs https://docs.npmjs.com/cli/v7/configuring-npm/package-json#license
so according to https://dev.handsontable.com/docs/next/documentation-license/ we should leave it as it is.